themes/gbe-theme/components/_products-sayt.scss

```diff
@@ -0,0 +1,125 @@
+    flex: var(--product-flex-width);
+    min-width: var(--product-sayt-min-width);
+
+    gbe-product {
+      border-right: var(--border-secondary);
+      height: var(--gbe-product-sayt-height);
+      position: relative;
+
+      .gbe-product-variants {
+        display: none;
+      }
+
+      a {
+       text-decoration: none;
+      }
+
+      a:visited {
+        h3 {
+          color: var(--gbe-product-description-visited-link-color);
+        }
+      }
+
+      img {
+        border-bottom: var(--gbe-product-img-border-bottom);
+        display: block;
+        height: var(--gbe-product-image-height);
+        margin: 0 auto;
+        max-width: var(--gbe-product-image-max-width);
+        padding: 20px 0;
+      }
+
+      h3, p, span {
+        padding-left: 15px;
+        font-family: var(--font-family-text);
+      }
+
+      span {
+        display: -webkit-box;
+        line-height: 1.5;
+        overflow: hidden;
+        -webkit-line-clamp: 3;
+        -webkit-box-orient: vertical;
+        color: var(--gbe-product-description-default-link-color);
+        font-size: var(--font-size-primary);
+      }
+
+      h3 {
+        box-sizing: border-box;
+        color: var(--gbe-product-description-default-link-color);
+        display: -webkit-box;
+        // font-family: var(--font-family-text);
+        font-size: var(--font-size-primary);
+        line-height: 1.5;
+        margin: 15px 15px 7px 0px;
+        overflow: hidden;
+        -webkit-line-clamp: var(--gbe-product-description-line-clamp);
+        -webkit-box-orient: vertical;
+      }
+
+      &:hover {
+        cursor: var(--cursor-type-pointer);
+
+        h3 {
+          color: var(--gbe-product-description-hover-link-color);
+          text-decoration: var(--gbe-product-description-active-hover-text-decoration);
+        }
+      }
+
+      &:active {
+        h3 {
+          color: var(--gbe-product-description-active-link-color);
+          text-decoration: var(--gbe-product-description-active-hover-text-decoration);
+        }
+      }
+
+      &:focus {
+        h3 {
+          color: var(--gbe-product-description-focus-link-color);
+        }
+      }
+
+      .gbe-price {
+        color: var(--font-color-secondary);
+        font-size: var(--gbe-product-price-font-size);
+        margin: 0;
+        margin: 0 0 10px 0px;
+      }
+    }
+  }
+
+  .gbe-product-tile-wrapper:nth-child(2) {
+    border-bottom-left-radius: var(--gbe-products-sayt-border-radius);
+    border-top-left-radius: var(--gbe-products-sayt-border-radius);
+  }
+
+  .gbe-product-tile-wrapper:nth-child(3) {
+    border-left: none;
+    border-right: none;
+  }
+
+  .gbe-product-tile-wrapper:nth-child(4) {
+    border-bottom-right-radius: var(--gbe-products-sayt-border-radius);
+    border-top-right-radius: var(--gbe-products-sayt-border-radius);
+  }
+
+  .gbe-product-tile-wrapper:nth-child(n + #{$num-of-tiles-desktop}) {
+    display: none;
```

Comment:
I was told to account for certain cases and hide the rest as per ryan

Reply:
I'm not saying that we should change this now because it can be left for later, but it's my opinion that a default theme should be able to support almost every (but ideally every) feature that the underlying components support. Maybe we can work on this in the future.